flow/designs/src/mock-alu/src/test/resources/README.md

```diff
@@ -0,0 +1,7 @@
+ASAP7 behavioral model files
+============================
+
+The multiplier is implemented in ASAP7 using Perhaps https://github.com/antonblanchard/vlsiffra
```

Comment:
@antonblanchard FYI, the multiplier from vlsiffra worked on the first try and passes unit-tests using the ASAP7 behavioral model :-)